docs/project_defn.rst

```diff
@@ -298,29 +299,28 @@ using `GCP Batch <https://cloud.google.com/batch>`_ and `Cloud Run <https://clou
        repository.
 *  ``job_environment``: Optional. Specifies the computing requirements for each simulation.
 
-    *  ``vcpus``: Number of CPUs to allocate for running each simulation. Default: 1.
-    *  ``memory_mib``: Amount of RAM memory needed for each simulation in MiB. Default: 1024.
-       For large multifamily buildings this works better if set to 2048.
+    *  ``vcpus``: Optional. Number of CPUs to allocate for running each simulation. Default: 1.
+    *  ``memory_mib``: Optional. Amount of RAM memory to allocate for each simulation in MiB.
+       For large multifamily buildings this works better if set to 2048. Default: 1024.
     *  ``boot_disk_mib``: Optional. Extra boot disk size in MiB for each task. This affects how
        large the boot disk will be (see the `Batch OS environment docs`_ for details) of the
        machine(s) running simulations (which is the disk used by simulations). This will likely need
        to be set to at least 2,048 if more than 8 simulations will be run in parallel on the same
        machine (i.e., when vCPUs per machine_type ÷ vCPUs per sim > 8). Default: None (which should
        result in a 30 GB boot disk according to the docs linked above).
-    *  ``machine_type``: GCP Compute Engine machine type to use. If omitted, GCP Batch will
+    *  ``machine_type``: Optional. GCP Compute Engine machine type to use. If omitted, GCP Batch will
        choose a machine type based on the requested vCPUs and memory. If set, the machine type
        should have at least as many resources as requested for each simulation above. If it is
        large enough, multiple simulations will be run in parallel on the same machine. Usually safe
        to leave unset.
-    *  ``use_spot``: true or false. This tells the project whether to use
-       `Spot VMs <https://cloud.google.com/spot-vms>`_ for data simulations, which can reduce
-       costs by up to 91%. Default: false
+    *  ``use_spot``: Optional. Whether to use `Spot VMs <https://cloud.google.com/spot-vms>`_
```

Comment:
maybe we should change the default to true here? thoughts?

Reply:
I think I prefer a default of False, because 1) that's the default when creating GCP jobs directly and 2) I think it's best for the default to be the most reliable option, at least as long as using spot instances at scale continues to require manual retries sometimes.